addons/optics/XEH_postInit.sqf

```diff
@@ -6,6 +6,12 @@ if (!hasInterface) exitWith {};
 GVAR(camera) = objNull;
 
 0 = 0 spawn {
+    // Workarround for the ThreeDen Editor Camera
+    //ToDo: verify if command will exist in 1.56 release
+    if (((productVersion select 2) >= 155) && {is3DEN}) exitWith {
```

Comment:
Maybe `!isNil {is3DEN}` instead? Just in case they don't finish the editor for 1.56.